targets/TARGET_STM/TARGET_STM32L4/TARGET_STM32L486xG/TARGET_NUCLEO_L486RG/USBHALHost_STM_TARGET.h

```diff
@@ -0,0 +1,19 @@
+/* Copyright (c) 2016 mbed.org, MIT License
```

Comment:
Out of curiosity, why are these new files MIT license compare to apache 2.0 that is being used or BSD as its in the current folder?